src/status_im2/contexts/communities/menus/request_to_join/view.cljs

```diff
@@ -73,60 +74,72 @@
     :render-fn                         community-rule-item}])
 
 (defn request-to-join
-  [{:keys [permissions name id images
-           can-join? can-request-access?
+  [{:keys [permissions
+           name
+           id
+           images
+           can-join?
+           can-request-access?
            requested-to-join-at]}]
-  (let [agreed-to-rules? (reagent/atom false)
-        is-open?         (not= 3 (:access permissions))]
-    (fn []
-      [rn/scroll-view {:style style/page-container}
-       [rn/view
-        {:style style/title-container}
+  (let [agreed-to-rules? (reagent/atom false)]
+    [safe-area/consumer
+     (fn [insets]
+       [:f>
+        (fn []
+          (let [{window-height :height} (rn/use-window-dimensions)
+                is-open?                (not= 3 (:access permissions))]
+            [rn/scroll-view {:style {:max-height (- window-height (:top insets))}}
```

Comment:
i feel like something wrong is here, why content should care about max height? bottom sheet should care about this cc @ibrkhalil

Reply:
also im not sure about `window-height` , bottom sheet should never go so hight , probably it should be a modal screen then ?